physics/GFS_MP_generic_post.F90

```diff
@@ -121,6 +134,52 @@ subroutine GFS_MP_generic_post_run(
       do i = 1, im
         rain(i) = rainc(i) + frain * rain1(i) ! time-step convective plus explicit
       enddo
+!
+! Combine convective reflectivity with MP reflectivity for selected
+! parameterizations.
+     if ( (imp_physics==imp_physics_thompson .or. imp_physics==imp_physics_nssl) .and. &
+       (imfdeepcnv==imfdeepcnv_samf .or. imfdeepcnv==imfdeepcnv_gf .or. imfshalcnv==imfshalcnv_gf) ) then
+         do i=1,im
+           factor(i) = 0.0
+           lfrz = .true.
+           zfrz(i) = phil(i,1) / con_g 
+           do k = levs, 1, -1
+             zo(i,k) =  phil(i,k) / con_g 
+             if (gt0(i,k) >= 273.16 .and. lfrz) then
+              zfrz(i) = zo(i,k)
+              lfrz = .false.
+             endif
+           enddo
+         enddo
+!
```

Comment:
Line 132 defined the variable: onebg = one/con_g, so lines 145 and 147 can be replaced by multiplication since multiplication is faster than division for some processors